agenda-webapps/src/main/webapp/WEB-INF/conf/agenda/templates/notification/web/VoteWebPlugin.gtmpl

```diff
@@ -0,0 +1,22 @@
+<% eventTitle = _ctx.escapeHTML(eventTitle); %>
+<li class="<%= READ %> clearfix" data-id="<%= NOTIFICATION_ID %>">
+    <div class="media agendaWebNotifContent">
+        <div class="avatarXSmall pull-left">
+            <img class="spaceAvatar" src="<%= calendarOwnerAvatarUrl %>"/>
+        </div>
+        <div class="media-body">
+            <div class="contentSmall" data-link="<%= eventURL %>">
+                <div class="status">
+                    <%
+                    String eventTitle = "<span style=\"color: #578dc9; text-decoration: none;font-weight:bold; font-family: 'HelveticaNeue Bold', Helvetica, Arial, sans-serif\" >" + eventTitle + "</span>";
+                    String agendaName = "<a style=\"text-decoration: none; font-weight:bold;font-family: 'HelveticaNeue Bold', Helvetica, Arial, sans-serif\" >" + agendaName +"</a>";
+                    String participantName = "<a style=\" text-decoration: none; font-weight:bold;font-family: 'HelveticaNeue Bold', Helvetica, Arial, sans-serif\" >" + participantName +"</a>";
+                    %>
+                    <%= _ctx.appRes("Notification.agenda.event.mail.body.date.vote", participantName, eventTitle, agendaName) %>
```

Comment:
The I18N key isn't good, you used `Notification.agenda.event.mail.body.date.vote`

Reply:
the content of displayed message is the same `{0} voted for a date for the following event {1} in the {2} space`